documentation/book/con-deploying-cluster-operator-using-operator-hub.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// assembly-cluster-operator.adoc
+
+[id='deploying-cluster-operator-using-operator-hub-{context}']
+= Deploying the Cluster Operator using OperatorHub.io
+
+{OperatorHub} is a catalog of Kubernetes operators.
+Operator Hub is using {OperatorLifecycleManager} to install and manage Kubernetes operators.
```

Comment:
About "Operator Hub", why using an attribute in the sentence before but pure text here?